core/modules/modDistributionList.class.php

```diff
@@ -248,9 +248,14 @@ public function __construct($db)
 		$this->rights[$r][5] = 'read'; // In php code, permission will be checked by test if ($user->rights->distributionlist->level1->level2)
 		$r++;
 		$this->rights[$r][0] = $this->numero + $r; // Permission id (must not be already used)
-		$this->rights[$r][1] = 'DistributionListPermCreateUpdate'; // Permission label
+		$this->rights[$r][1] = 'DistributionListPermCreate'; // Permission label
 		$this->rights[$r][4] = 'distributionlist'; // In php code, permission will be checked by test if ($user->rights->distributionlist->level1->level2)
-		$this->rights[$r][5] = 'write'; // In php code, permission will be checked by test if ($user->rights->distributionlist->level1->level2)
+		$this->rights[$r][5] = 'create'; // In php code, permission will be checked by test if ($user->rights->distributionlist->level1->level2)
+		$r++;
+		$this->rights[$r][0] = $this->numero + $r; // Permission id (must not be already used)
```

Comment:
Tous les nouveaux droits doivent gérer leurs numéros avec une concaténation plutôt qu'une addition pour éviter des problèmes
```suggestion
		$this->rights[$r][0] = $this->numero . $r; // Permission id (must not be already used)
```